diff --git a/src/app/pages/solution/lifecycle/lifecycle.component.html b/src/app/pages/solution/lifecycle/lifecycle.component.html
index d93a60686..71f7920c7 100644
--- a/src/app/pages/solution/lifecycle/lifecycle.component.html
+++ b/src/app/pages/solution/lifecycle/lifecycle.component.html
@@ -1,4 +1,4 @@
-<div class="panel panel--ltgray panel--bordered" #lifecyclePanel [class.panel--bottom-padding]="panelBottomPaddingNeeded">
+<div class="panel panel--ltgray panel--bordered" #lifecyclePanel [class.panel--bottom-padding]="panelBottomPaddingNeeded || panelBottomPaddingNeededForMessage">
 	<div class="row base-padding-top">
 		<div class="col-md-3 base-padding-left" *ngIf="!status.loading.racetrack && componentData?.racetrack?.stage">
 			<ng-container *ngTemplateOutlet="racetrack"></ng-container>
diff --git a/src/app/pages/solution/lifecycle/lifecycle.component.ts b/src/app/pages/solution/lifecycle/lifecycle.component.ts
index 50ec92e5d..04a2116a3 100644
--- a/src/app/pages/solution/lifecycle/lifecycle.component.ts
+++ b/src/app/pages/solution/lifecycle/lifecycle.component.ts
@@ -134,6 +134,7 @@ export class LifecycleComponent implements OnDestroy {
 	public atxScheduleCardOpened = false;
 	public recommendedAtxScheduleCardOpened = false;
 	public panelBottomPaddingNeeded = false;
+	public panelBottomPaddingNeededForMessage = false;
 	public sessionSelected: AtxSessionSchema;
 	public customerId: string;
 	public buId: string;
@@ -1027,10 +1028,10 @@ export class LifecycleComponent implements OnDestroy {
 
 			if (this.calculatePercentage(this.componentData.racetrack.pitstop) === 1) {
 				this.showCompletionPopup = true;
-				this.panelBottomPaddingNeeded = true;
+				this.panelBottomPaddingNeededForMessage = true;
 				setTimeout(() => {
 					this.showCompletionPopup = false;
-					this.panelBottomPaddingNeeded = false;
+					this.panelBottomPaddingNeededForMessage = false;
 				}, this.timeout);
 			}
 			// Need to call getRacetrackInfo to get the latest data
